server/text_generation_server/models/custom_modeling/flash_mistral_modeling.py

```diff
@@ -487,10 +486,10 @@ def __init__(self, prefix: str, config, weights, name=None):
             ),
             weights=weights,
         )
-        self.max_past = config.sliding_window
+        self.max_past = config.sliding_window if SUPPORTS_WINDOWING else None
         self.max_past_tensor = (
-            torch.tensor(config.sliding_window, device=weights.device)
-            if self.max_past is not None
+            torch.tensor(self.max_past, device=weights.device)
```

Comment:
Doesn't this try to create a tensor with the `None` value if windowing is not supported? Same below.

Reply:
ahh yes you are right 😅, fixed in the latest changes to avoid this all together